### PR TITLE
Add MarshalManyPayloadWithoutIncluded

### PR DESCRIPTION
**Pull request by lulezi**
`MarshalManyPayloadWithoutIncluded` does exactly the same as `MarshalOnePayloadWithoutIncluded` but for a `MarshalManyPayload`.

### CODE REVIEW

**Comment by lulezi** [comment]
I signed it!

**Comment by genexp** [comment]
Nice!!

File: response.go [hunk 1]
```diff
@@ -85,6 +85,37 @@ func MarshalOne(model interface{}) (*OnePayload, error) {
...
+		}
+
+		rootNodes = append(rootNodes, rootNode)
+	}
+
+	if err := json.NewEncoder(w).Encode(&ManyPayload{Data: rootNodes}); err != nil {
+		return err
+	}
+
+	return nil
```
Comment (aren55555) [inline response.go]: I would change the implemenation of this function to:

```go
func MarshalManyPayloadWithoutIncluded(w io.Writer, models interface{}) error {
	m, err := convertToSliceInterface(&models)
	if err != nil {
		return err
	}
	payload, err := MarshalMany(m)
	if err != nil {
		return err
	}

	// Empty the included
	payload.Included = []*Node{}

	if err := json.NewEncoder(w).Encode(payload); err != nil {
		return err
	}

	return nil
}
```

Such that the `MarshalMany` code path is still used.

Comment (aren55555) [inline response.go]: @shwoodard

**Pull request merged**